frontend/Makefile

```diff
@@ -1,6 +1,26 @@
-.PHONY: all
+.PHONY: all test clean
 
-all: cfg_pb2.py
+PROTO_DIR = src/frontend/proto
+TEST_DIR = src/frontend/cfg_generator
+VPATH = $(PROTO_DIR)
+PROTOS = cfg.proto
+PY_PROTOS = $(patsubst %.proto, %_pb2.py, $(PROTOS))
+PYI_PROTOS = $(addsuffix i, $(PY_PROTOS))
 
-cfg_pb2.py: 
-	protoc --python_out=./src/frontend_benchmark/proto/ --mypy_out=./src/frontend_benchmark/proto/ ./cfg.proto
+PY_TESTS = dfs_chase_gen_test.py \
+	   inst_pointer_chase_gen_test.py
+
+%_pb2.py: %.proto
+	protoc --python_out=. --mypy_out=. $^
+
+all: $(PY_PROTOS)
+
+test: all
```

Comment:
Should we add a mypy check here too?

Reply:
We can add a pylint and mypy check here in a future commit.
I also need to adjust make test to run the new tests/use pylint.